Bdd/features/environment.py

```
@@ -1,7 +1,44 @@
+
+    # Restore syslog-ng config if it was changed during the scenario
+    if hasattr(context, "syslog_ng_config_changed") and context.syslog_ng_config_changed:
+        shutil.copy(SYSLOG_NG_FULL_CONF, SYSLOG_NG_CONF)
+        try:
+            with socket.socket(socket.AF_UNIX, socket.SOCK_STREAM) as sock:
+                sock.connect(SYSLOG_NG_CTL)
+                sock.sendall(b"RELOAD\n")
+                sock.recv(1024)
+            time.sleep(0.5)
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)